include/core/map.h

```diff
@@ -14,6 +14,13 @@
 #include <QGraphicsPixmapItem>
 #include <math.h>
 
+#define DEFAULT_BORDER_WIDTH 2
+#define DEFAULT_BORDER_HEIGHT 2
+
+// Number of metatiles to draw out from edge of map. Could allow modification of this in the future.
+// porymap will reflect changes to it, but the value is hard-coded in the projects at the moment
+#define BORDER_DISTANCE 7
```

Comment:
Why 7, and not 6?  6 is what emerald/ruby use.

Reply:
Drawing 7 shows the maximum extent of the player view. If you only draw 6 the player view rectangle extends beyond the edge of what's drawn.